scss/main.scss

```diff
@@ -90,10 +90,28 @@ $animation-length: 80s;
   background-color: #fff !important;
   box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.05);
   transition: background-color 200ms linear;
+
   .nav-link {
     color: $lavender !important;
     transition: color 200ms linear;
   }
+    
+  .logo {
+    background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
+    width: 50px;
+    height: 50px;
+    border-radius: 50px;
```

Comment:
lets add a media query here for 767px to decrease the size of the orb to match with the decreased font size of the navbar links